mautic-inc/plugin-integrations
# MAUT-2935 - Added new subscriber and events for integrations to add okens for mapped integration objects

This code PR is required for https://github.com/mautic-inc/plugin-salesforce/pull/78

Testing Steps:

1. Make sure the code from the above PR is also available
2. Install and configure the Salesforce plugin
3. Import some contacts or leads from Salesforce into Mautic
4. Create a new email and in the email builder, go to the token list
5. There should be the option for a Salesforce Lead and Salesforce Contact Link tokens
6. Add one of these to them email, change any of the defaults you like, like the link text
7. Send the email and verify that the token populates properly. It should create a link that goes to the Salesforce page representing that contact / lead.
